ui/src/types/logs.ts

```diff
@@ -166,3 +166,14 @@ export enum Operator {
   EQUAL = '==',
   NOT_EQUAL = '!=',
 }
+
+export enum MatchType {
+  NONE = 'no-match',
```

Comment:
personally i prefer these capitalized like MatchType.`None` -- what's being done elsewhere on the whole?

Reply:
Looked into this. I see usage of `thing`, `THING`, and `Thing`. The capitalized camel casing seems common enough.

Reply:
Going to leave this as is since I don't want to make this file inconsistent. It might be nice to add a linting rule around this.